main.cpp

```diff
@@ -231,11 +242,9 @@ void draw()
 
     cs::shader->use();
 
-    cs::shader->setProjectionMatrix(
-            glm::perspective(glm::radians(90.0f), (float) WINDOW_WIDTH / (float) WINDOW_HEIGTH,
-                             0.1f, 200.0f));
-    cs::shader->setViewMatrix(
-            glm::lookAt(glm::vec3(0, 0.5f, 2.0f), glm::vec3(0, 0, 0), glm::vec3(0, 1, 0)));
+    cs::shader->setProjectionMatrix(MATRIX_PROJECTION);
```

Comment:
Kan MATRIX_PROJECTION en MATRIX_VIEW hernoemt worden met CONFIG_ prefix

Kan er een voorbeeld worden toegevoegd van deze config waardes in user-config-example.h.txt